astropy/io/fits/convenience.py

```diff
@@ -420,8 +428,8 @@ def writeto(filename, data, header=None, output_verify='exception',
     hdu = _makehdu(data, header)
     if hdu.is_image and not isinstance(hdu, PrimaryHDU):
         hdu = PrimaryHDU(data, header=header)
-    hdu.writeto(filename, clobber=clobber, output_verify=output_verify,
-                checksum=checksum)
+    hdu.writeto(filename, clobber=clobber, overwrite=overwrite,
```

Comment:
That won't work, since all handling of `clobber` versus `overwrite` is done in the `_File` class. And `hdu.writeto` implicitly calls that, so parameters need to be passed through.